addons/mail/static/src/components/composer_text_input/composer_text_input.scss

```diff
@@ -5,6 +5,12 @@
 .o_ComposerTextInput {
     min-width: 0;
     position: relative;
+
+    // When composer is not compact, all textarea should be rounded so it
+    // does not interfer with outer borders
+    &:not(.o-composer-is-compact) .o_ComposerTextInput_textarea {
```

Comment:
`textarea` part should have modifier, because this rulesets add 1 level of specificity.

Reply:
That means we have to pass the "isCompact" prop from composer to composer text input. Is that better ?

Reply:
`ComposerTextInput` is an arbitrary split of `Composer` dedicated to the main text input. This is quite similar to `ThreadViewer` and `MessageList`. I don't mind having almost same props, because the sole component user of `ComposerTextInput` is `Composer` itself.